cves/kernel/CVE-2014-1690.yml

```diff
@@ -395,37 +418,42 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies:
+    applies: false
     note:
   least_privilege:
-    applies:
+    applies: false
     note:
   frameworks_are_optional:
-    applies:
-    note:
+    applies: true
+    note: |
+      This vulnerability came out of a framework -- it stemmed from the 
+      inclusion of NetFilter Connection Tracking (NF_CONNTRACK) support 
+      for IRC protocol (NF_NAT_IRC).
   native_wrappers:
-    applies:
+    applies: false
     note:
   distrust_input:
-    applies:
-    note:
+    applies: true
+    note: This is a common mitigation tactic for buffer overflows.
   security_by_obscurity:
-    applies:
+    applies: false
     note:
   serial_killer:
-    applies:
+    applies: false
     note:
   environment_variables:
-    applies:
+    applies: false
     note:
   secure_by_default:
-    applies:
+    applies: false
     note:
   yagni:
-    applies:
-    note:
+    applies: true
+    note: |
+      Applies as the changes made (to support IPv6) were unnecessary 
```

Comment:
Why were the changes to support IPv6 unnecessary ?

Reply:
Opinionated -- IPv6 isn't the dominant IP version out today so there's no serious or code-threatening push to switch over to it now.  I agree, though, as this was an improvement to become more inclusive to anyone USING IPv6 and thus may be more important than I had originally considered.